controller/remote/controller.go

```diff
@@ -35,6 +35,12 @@ const (
 	serveCommandName = "_INTERNAL_SERVE"
 )
 
+var (
+	defaultLogFilename    = fmt.Sprintf("buildx.%s.log", version.Revision)
+	defaultSocketFilename = fmt.Sprintf("buildx.%s.sock", version.Revision)
+	defaultPIDFilename    = fmt.Sprintf("buildx.%s.pid", version.Revision)
```

Comment:
nit: Should be encoded not to include "/" (e.g. sha256?)

Reply:
I *think* these revisions are already sha256-encoded, they're derived from the git revision.

@crazy-max does this make sense to you? Is there a different version identifier that might make more sense here / do we need to add a round of sha256 to remove special characters?

Reply:
There is an edge case where `version.Revision` can be empty if not built from the Dockerfile. I think we could use the same pattern as the kubernetes driver with `contextPathHash` if that makes sense: https://github.com/docker/buildx/blob/74f64f88a7a94af52d4e64530d79bf0303524fbf/controller/build/build.go#L182-L186

Reply:
As discussed I got confused as this is for buildx server and not build commands. We could use https://pkg.go.dev/runtime/debug#ReadBuildInfo and read `vcs.revision`.

Reply:
Added a follow-up in https://github.com/docker/buildx/pull/1637.